voyager-transitions/src/commonMain/kotlin/cafe/adriel/voyager/transitions/ScreenTransition.kt

```diff
@@ -55,6 +55,26 @@ public fun ScreenTransition(
     )
 }
 
+@Composable
+public fun ScreenTransition(
+    navigator: Navigator,
+    defaultTransition: ScreenTransition,
+    modifier: Modifier = Modifier,
+    content: ScreenTransitionContent = { it.Content() }
+) {
```

Comment:
Can you mark this composable as @ExperimentalVoyagerApi/

Reply:
done